konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/registry/MicrometerRegistry.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *  ******************************************************************************
+ *  * Copyright (c) 2020 Konduit K.K.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Apache License, Version 2.0 which is available at
+ *  * https://www.apache.org/licenses/LICENSE-2.0.
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  * License for the specific language governing permissions and limitations
+ *  * under the License.
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *  *****************************************************************************
+ */
+package ai.konduit.serving.pipeline.registry;
+
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+
+public class MicrometerRegistry {
+    public static io.micrometer.core.instrument.MeterRegistry getRegistry() {
+        return io.micrometer.core.instrument.Metrics.globalRegistry.add(new SimpleMeterRegistry());
```

Comment:
We shouldn't be adding a new registry on every call.

```
public class MicrometerRegistry {
    static {
        io.micrometer.core.instrument.Metrics.globalRegistry.add(new SimpleMeterRegistry());
    }
    public static io.micrometer.core.instrument.MeterRegistry getRegistry() {
        return io.micrometer.core.instrument.Metrics.globalRegistry;
    }
}
```

Reply:
Fixed.